web/helper.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Helper</title>
+</head>
+
+<body>
+   <script>
+      // Request local storage from bookmarklet on Infinite Craft
+      window.parent.postMessage('requestLocalStorage', 'https://neal.fun');
+
+      // Add event listener for reciving local storage from Infinite Craft
+      window.addEventListener('message', function (event) {
```

Comment:
Basically just your bookmarklet code but moved into an event listener that gets the local storage data from the current bookmarklet.

I was on the fence about if this is something we wanted to do or not for a while, since it make it more complicated than it strictly needs to be, however, once people are actively using this we really don't want to make a bookmark just stop working out of nowhere with no way to update anything. So by embedding an iframe that we have complete control over we can give ourselves chances to change anything we want about the way the data is serialized and sent over. And if we do ever need to fully retire this to have them update it, we can always just change the redirect link here to point towards a page telling them to do so, which is very convenient.